pipeline/trusted-resources/pkg/config/store.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"context"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/reconciler"
+)
+
+type cfgKey struct{}
+
+// +k8s:deepcopy-gen=false
```

Comment:
Just out of curiosity :) why do we need to explicitly disable the deepcopy-gen?

Reply:
This is from https://github.com/tektoncd/chains/blob/5586b90ef1c108019a6a91d71156367a449f281e/pkg/config/store.go#L28-L31

My guess is we don't want to generate deepcopy for this

Reply:
and this kind of store type contains all pointer fields, maybe that's one reason?